components/Tabs/Tabs.js

```diff
@@ -59,4 +67,8 @@ class TabItem {
 
 */
 
-links = document.querySelectorAll();
\ No newline at end of file
+const links = document.querySelectorAll('.tabs-link');
+links.forEach((link) => {
+  return new TabLink(link) 
+});
```

Comment:
There's nothing wrong with doing things this way but it could be done in one line, `.forEach` is changed off an array and since `querySelectorAll` returns an array it could be written like this `querySelectorAll('something').forEach()`, also instead of having a function that has a just a return statement, the arrow function will return what's in front of it implicitly if it doesn't see curly braces so you could do something like this `forEach(something => something * 3)`. with this in mind the 3 lines above could become `document.querySelectorAll(.tabs-link).forEach(link => new TabLink(link))`